app/frontend/src/styles/application/form.scss

```diff
@@ -19,3 +19,30 @@
   color: $govuk-link-hover-colour;
   text-decoration: underline;
 }
+
```

Comment:
i would be inclined to make the naming here a bit more BEM like and abstract rather than specifically `starts asap`

Reply:
maybe `clear-form` ?